build.sh

```diff
@@ -87,19 +97,25 @@ if [ $# -eq 0 ]; then
 fi
 
 function runTests() {
-    echo "$(date +%T) running tests"
+    local timestamp=$1
+    startJetty $timestamp
+    echo "$${timestamp} running tests"
```

Comment:
Ok, I think this should be the last thing (it wasn't a spelling typo so my dictionary didn't help me out here..), sorry for making you go back and forth here.

There's an extra `$` before the timestamp. This results in something like:

```
16:00:37 jetty server up and running on pid 595959
595716{timestamp} running tests
```
instead of something like:
```
16:01:20 jetty server up and running on pid 596437
20210113160105 running tests
```

Reply:
very nice!, I've not noticed it, corrected